lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -143,12 +155,34 @@ for barclamp in $deactivation_list; do
 done
 
 
+### Stop chef-clients at the nodes, to ensure services shutdown in next step won't be started by chef
+for node in $allocated_nodes; do
+  echo "Stopping chef-client service on ${node}..."
+  ssh "$node" 'rcchef-client stop'
+done
+
+
+### Stop pacemaker resources in all clusters and put the nodes into standby mode
+echo_summary "Stoping pacemaker resources..."
+
+for founder in $cluster_founders; do
+  echo "Stopping crm resources on ${founder}..."
+  ssh "$founder" '
+    for type in clone group ms primitive; do
+      for resource in $(crm configure show | grep ^$type | cut -d " " -f2); do
+        crm resource stop $resource
+      done
+    done
+    '
+done
+
+
 ### Stop openstack services
 echo_summary "Stopping OpenStack services..."
 
 for node in $allocated_nodes; do
   echo "Disabling OpenStack services on ${node}..."
-  ssh "$node" 'rcchef-client stop; for i in /etc/init.d/openstack-* /etc/init.d/openvswitch-switch ; do if test -e $i ; then initscript=`basename $i`; chkconfig -d $initscript; $i stop; fi; done'
+  ssh "$node" 'for i in /etc/init.d/openstack-* /etc/init.d/openvswitch-switch /etc/init.d/ovs-usurp-config-* ; do if test -e $i ; then initscript=`basename $i`; chkconfig -d $initscript; $i stop; fi; done'
```

Comment:
I think we need to add openais and drbd to the list of services to stop and disable: this fixes a small error in the logs when re-applying the barclamp where pacemaker doesn't initially see the drbd resources as stopped. It also feels safer to me anyway (as we don't want pacemaker to do anything during the rest of the upgrade process).